tests/test_pango.py

```diff
@@ -0,0 +1,124 @@
+from manim import PangoText, START_X, START_Y, SVGMobject
+import os
+import cairocffi
+import pangocairocffi
+import pangocffi
+
+rtl_text = """صباح الخير
+مرحبا جميعا"""
+
+width = 600
+height = 400
+
+
+def test_general_text_svgobject():
+    """Checks number of submobjects generated when directly called using ``SVGMobject``"""
+    text = "hello"
+    size = 1
+    folder = os.path.abspath(os.path.join("media", "texts"))
+    if not os.path.exists(folder):
+        os.makedirs(folder)
+    filename = os.path.join(folder, "hello.svg")
```

Comment:
Can you please make a note somewhere (perhaps a new issue) to get rid of this as soon as SceneFIleWriter and TexFileWriter are refactored? This shouldn't be necessary...

Reply:
@behackl already knows this issue and discussed on discord. Also why #476 is pending.

Reply:
I understand that, but what I mean to say is that the real fix to #476 is to refactor SceneFileWriter. Once that is done, we should change this test. That's all I'm saying.